src/profile/v100/address/wrap_ret_address.rs

```diff
@@ -218,6 +227,14 @@ macro_rules! decl_ret_wrapped_address {
                     let sut = SUT::sample();
                     assert_eq!([<$address_type:snake _address_map_to_network>](&sut, sut.network_id()), sut); // unchanged
                 }
+
+                #[test]
+                fn random_address() {
+                    for network_id in NetworkID::all() {
+                        let sut = [<new_ $address_type:snake _address_sample_random>](network_id);
+                        assert_eq!(SUT::from_str(&sut.to_string()).unwrap(), sut); // unchanged
```

Comment:
add a comment that the string roundtrip ensures that we have used to correct entity => correct Bech32 HRP, thus ensuring correctness of `EntityType`

I'd also like a `let n = 1000'; ... hashset.len() == n` test, ensuring we are indeed using randomness as the remamning 29 bytes in the node id.